### PR TITLE
FFM-9032 Bump JavaScript SDK version to 1.15.0

### PR DESCRIPTION
**Pull request by erdirowlands**
# What
 Bump JavaScript SDK version from 1.13.0 to 1.15.0 - see release notes for that version https://github.com/harness/ff-javascript-client-sdk/releases/tag/1.15.0 

# Testing 
Smoke tests on the sample application:
- toggle flags for all flag types, they all update
- turn internet off, observe new SSE disconnect messages, turn it back on 10 seconds later and it reconnects and stream toggles as normal

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
+    "declaration": true
```
Comment (erdirowlands) [inline tsconfig.json:18]: I had to add this declaration property for the sample application - it was complaining it couldn't get types. Not sure on the wider implications for release.

Comment (erdirowlands) [reply tsconfig.json]: Removed now @knagurski

File: tsconfig.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "react-jsx",
```
Comment (knagurski) [inline tsconfig.json:17]: Trailing comma makes JSON sad

Comment (erdirowlands) [reply tsconfig.json]: Fixed

**Review by knagurski — APPROVED** [review]
Best. PR. EVER!

**Pull request merged**